index.ts

```diff
@@ -462,8 +464,12 @@ const getRepoPath = (url: URL | Location = location): string | undefined => {
 	return undefined;
 };
 
+/** Get the 'user/repo' part from an URL */
+const getRepoURL = (url: URL | Location = location): string => url.pathname.slice(1).split('/', 2).join('/');
+
```

Comment:
Can you import the tests for this function from refined GitHub as well?

Reply:
There aren't any (or I missed them) :slightly_frowning_face: